interfaces/builtin/fwupd_test.go

```diff
@@ -175,6 +184,26 @@ func (s *FwupdInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	c.Assert(dbusSpec.SecurityTags(), HasLen, 0)
 }
 
+func (s *FwupdInterfaceSuite) TestMountPermanentSlot(c *C) {
+	tmpdir := c.MkDir()
+	dirs.SetRootDir(tmpdir)
+
+	c.Assert(os.MkdirAll(filepath.Join(tmpdir, "/boot"), 0777), IsNil)
+
+	// On all-snaps systems, boot partition should be
+	// bind mounted from the host system if they exist.
```

Comment:
Same nit-pick for the comment here (it isn't so much about all-snaps systems as about app-provided slot).